h2o-core/src/main/java/water/webserver/H2OHttpViewImpl.java

```diff
@@ -23,6 +24,8 @@
  * This is intended to be a singleton per H2O node.
  */
 public class H2OHttpViewImpl implements H2OHttpView {
+  private static final Logger LOG = Logger.getLogger(H2OHttpViewImpl.class);
+  public static String DISABLE_NON_LEADER_API = H2O.OptArgs.SYSTEM_PROP_PREFIX + "disableNonLeaderApi";
```

Comment:
This is a system-level property, not directly user facing. One can always do -Dsys.ai.h2o.disableapinonleader=true, but users should not care about it. Therefore not taking it into H2o.args.